modules/op-release-notes-1-1.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-1_{context}"]
+= Release notes for Red Hat {pipelines-title} Technology Preview 1.1
+
+[id="new-features-1-1_{context}"]
+== New features
+{pipelines-title} Technology Preview (TP) 1.1 is now available on {product-title} {product-version}. {pipelines-title} TP 1.1 is updated to support:
+
+* Tekton Pipelines 0.14.3
+* Tekton `tkn` CLI 0.11.0
+* Tekton Triggers 0.6.1
+* ClusterTasks based on Tekton Catalog 0.14
+
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.1.
+
+=== Pipelines
+* Workspace support for VolumeClaimTemplates has been added. The VolumeClaimTemplate for a PipelineRun and TaskRun can now be added as a volume source for Workspaces. The tekton-controller then creates a `PersistentVolumeClaim` using the template that is seen as a `PersistentVolumeClaim` for all TaskRuns in the Pipeline. Thus you do not need to define the `PersistentVolumeClaim` configuration every time it binds a workspace that spans multiple tasks.
+* Support for improving audits:
+** The `PipelineRun.Status` field now contains the status of every `TaskRun` and the Pipeline specification used to instantiate a `PipelineRun` to monitor the progress of the `PipelineRun`.
+** Pipeline results are now added to the pipeline specification and `PipelineRun` status.
+** The `TaskRun.Status` field, now contains the exact Task specification used to instantiate the `TaskRun`.
+* Support to find the name of the PersistentVolumeClaim when a VolumeClaimTemplate is used as a volume source is now added using variable substitution.
```

Comment:
Should this be as a *sub* item of the one in line 20 (as it's both on `volumeClaimTemplate`)